shell/platform/darwin/macos/framework/Source/FlutterAppDelegate_Internal.h

```diff
@@ -0,0 +1,22 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_FLUTTERAPPDELEGATE_INTERNAL_H_
+#define FLUTTER_FLUTTERAPPDELEGATE_INTERNAL_H_
+
+#include "embedder.h"
```

Comment:
```suggestion
#include "flutter/shell/platform/embedder/embedder.h"
```

Reply:
It's actually not needed at all. Removed.